packages/core/src/App/Containers/RealAccountSignup/signup-error-content.jsx

```diff
@@ -34,7 +34,7 @@ const SignupErrorContent = ({ message, code, onConfirm }) => {
                 return (
                     <p>
                         <Localize
-                            i18n_default_text="An account with your details already exists. If you're having trouble signing in, let us know <0>chat</0>"
+                            i18n_default_text="An account with your details already exists.<br /> If you're having trouble signing in, let us know <0>chat</0>"
```

Comment:
Not sure if it's a correct way of adding new line

Reply:
@shayan-binary why not just have two separate localize components with a br tag in the middle?

Reply:
Or we can use `<p>` as component tags, within the same `<Localize />` component to facilitate the line breaks or paragraphing.

Reply:
fixed it as you mentioned. thanks